src/media/js/site/components/login.js

```diff
@@ -32,7 +32,7 @@ export class LoginButton extends React.Component {
                     Url(this.props.authUrl).q({signup: true}) :
                     this.props.authUrl;
 
-    const w = 320;
+    const w = 480;
```

Comment:
FWIW, this was an FxA bug that was fixed on their prod servers, but not dev.

# Dev

<img width="432" alt="screen shot 2015-10-06 at 1 51 27 pm" src="https://cloud.githubusercontent.com/assets/23885/10320587/7538dd36-6c31-11e5-81d0-f0e5adb6accb.png">

# Dev

<img width="432" alt="screen shot 2015-10-06 at 1 52 23 pm" src="https://cloud.githubusercontent.com/assets/23885/10320595/7e23465c-6c31-11e5-85f3-711476191e89.png">